subsys/bluetooth/host/iso.c

```diff
@@ -105,7 +105,11 @@ void hci_iso(struct net_buf *buf)
 
 	BT_ISO_DATA_DBG("buf %p", buf);
 
-	BT_ASSERT(buf->len >= sizeof(*hdr));
+	if (buf->len < sizeof(*hdr)) {
+		LOG_ERR("Invalid HCI packet size (%u)", buf->len);
```

Comment:
Since `LOG_ERR` does not log the function name by default, maybe it would be nice have something like
```suggestion
		LOG_ERR("Invalid HCI ISO packet size (%u)", buf->len);
```
and similar for the ACL packet, so that the user can know which type of packet has the incorrect size

Reply:
Great suggestion. Applied.

Reply:
@carlescufi Did you forget to save? :D I don't see anything being applied